packages/node/README.md

```diff
@@ -0,0 +1,28 @@
+# adCAPTCHA React
+
+This package provides a Node component for adCAPTCHA.
+
+## Installation
+
+```bash
+npm install @adcaptcha/node
+```
+
+## Usage
+
+```jsx
+import { verify } from '@adcaptcha/react';
+
+try {
+  const response = await verify(apiKey, token)
```

Comment:
Could do with more docs on what is in the return object

Reply:
What is `apiKey`, where does it come from? tell the reader! Perhaps give example

Same for `token`